tldw_Server_API/tests/Agent_Client_Protocol/test_mcp_runners_llm.py

```
@@ -0,0 +1,295 @@
+        llm_caller=llm_caller,
+        tool_gate=tool_gate,
+        tools=MCP_TOOLS,
+        max_iterations=max_iterations,
+    )
+    return runner, transport, llm_caller, tool_gate, callback, cancel_event
+
+
+@pytest.mark.asyncio
+async def test_llm_driven_single_turn():
+    """LLM returns text immediately; verify single COMPLETION event."""
+    runner, transport, llm_caller, tool_gate, callback, _ = _make_runner(
```

Ruff (0.15.6)

[warning] Unpacked variable `llm_caller` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)